app/app.go

```diff
@@ -233,6 +248,10 @@ func (a *app) ProvisionDevice(
 		switch integration.Provider {
 		case model.ProviderIoTHub:
 			err = a.provisionIoTHubDevice(ctx, deviceID, integration)
+		case model.ProviderIoTCore:
+			err = a.provisionIoTCoreDevice(ctx, deviceID, integration, &iotcore.Device{
+				Status: iotcore.StatusDisabled,
```

Comment:
being curious why do we provision in the disabled state? how do we travel to enabled state?

Reply:
Devices are always provisioned in "Disabled" state, and then get accepted when you accept the authset by the SetDeviceStatus function.